src/services/analytics/events/overview.ts

```diff
@@ -68,4 +68,12 @@ export const OVERVIEW_EVENTS = {
     action: 'Open Safe Claiming App from widget',
     category: OVERVIEW_CATEGORY,
   },
+  MISSING_SIGNATURES: {
```

Comment:
Not the best name, you need to name the action, not the button.

Reply:
what about 
```suggestion
  OPEN_MISSING_SIGNATURES: {
```
?